### PR TITLE
Fix bug in "items" function

### LINKED ISSUE #101: Found a problem with the args of the Dictionary "items" function.
**Current Results**
in gpython
```python
>>>> d_list = {'a' : 1, 'b' : 2}
>>>> for k, v in d_list.items("hi"):
>>>>     print(k,':',v)
a : 1
b : 2
```
in python
```python
>>>> d_list = {'a' : 1, 'b' : 2}
>>>> for k, v in d_list.items("hi"):
>>>>     print(k,':',v)
Traceback (most recent call last):
  File "test.py", line 2, in <module>
    for k, v in d_list.items("hi"):
TypeError: items() takes no arguments (1 given)
```

https://github.com/go-python/gpython/blob/af17d7daed3e64897f592923022638280abc0627/py/dict.go#L30-L39

I think we should add an error to this part of the code.

### LINKED ISSUE #102: NotImplemented comparison generate TypeError
TypeError occurs when comparing NotImplemented value from execution result.

### Expected result (cpython)
```python
>>> range(0).__ne__(2) == NotImplemented
True
```

### Actual result 
```python
>>> range(0).__ne__(2) == NotImplemented
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
    FIXME line of source goes here
TypeError: "unsupported operand type(s) for ==: 'NotImplementedError' and 'NotImplementedError'"
```

### PR DESCRIPTION
**Pull request by Sungmin-Joo**
Fixes #101

### CODE REVIEW

File: py/tests/dict.py [hunk 1]
```diff
@@ -35,6 +36,7 @@
         assert v == "b"
     if k == "c":
         assert v == 5.5
+assertRaises(TypeError,a.items,'a')
```
Comment (corona10) [inline py/tests/dict.py]: nit Please follow PEP8 when you writing python code.

```python
assertRaises(TypeError, a.items, 'a')
```

**Comment by Sungmin-Joo** [comment]
@corona10 
Okay, I'm done with it right away.

**Comment by Sungmin-Joo** [comment]
@corona10 
I'm sorry, but can you check it again?

**Comment by corona10** [comment]
@Sungmin-Joo Next time, I recommend you to work on the sperate branch rather than master branch :)

**Pull request merged**